### PR TITLE
fix scroll issue

### LINKED ISSUE #1: BUG: fix the scroll bug

### PR DESCRIPTION
**Pull request by Kayron013**
Fixes #1 
Keys used to play the game will no longer cause the page to scroll

### CODE REVIEW

File: app/Controller.tsx [hunk 1]
```diff
@@ -7,6 +7,19 @@ type Props = {
 };
 
 export default function Controller({ controller }: Props): JSX.Element {
```
Comment (sunnyeyez123) [inline app/Controller.tsx:9]: Can you add a comment about what this code does? i remember you told me in stand up but i can't remember. thanks!

**Review by sunnyeyez123 — CHANGES_REQUESTED** [review]
nits

File: app/Controller.tsx (hunk 1 above)
Comment (Kayron013) [reply app/Controller.tsx:9]: Added a comment.

**Review by sunnyeyez123 — APPROVED** [review]
Looks good. Thanks for the update.

**Pull request merged**